.eslintrc.js

```diff
@@ -423,7 +411,6 @@ const reactRules = {
 	'react/boolean-prop-naming': ERROR,
 	'react/button-has-type': ERROR,
 	'react/default-props-match-prop-types': ERROR,
-	'react/forbid-component-props': TRANSITION_WARNING,
```

Comment:
This is definitely aspirational and floods the warnings. If we want to address this, we should turn this option on locally to work on it, but in the meantime it's just noise.